entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+npm install && \
+./wait-for-it.sh -s -q -t 0 database:5432 -- npm start
```

Comment:
The `wait-for-it` script clobbers the `PORT` environment variable, so the standup-bot HTTP server ends up listening on whatever port the script is waiting on.  There's [an open issue](https://github.com/vishnubob/wait-for-it/issues/23) about it, which hasn't been touched in a bit, unfortunately.

As a stop-gap, locally, I replaced the `wait-for-it` invocation with a naive sleep:

```
sleep 10 && npm start
```

That seemed to do the trick.  Using `wait-for-it` is definitely preferable, though!  I think I'll fork that repo and try to PR a fix.